.gitignore

```diff
@@ -1,2 +1,5 @@
 /target
 Cargo.lock
+
+# IntelliJ IDEA files
+/.idea
```

Comment:
:-1:

Reply:
Why shouldn't we ignore the IDEA directory?

Reply:
Ohh my bad, I read it as it was included =) Yea, we should defo ignore it...